### ISSUE TITLE
Settings UI: Shift+tab focuses .settings-tree-focus-sync when it should focus the ToC

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#94000]
Repro:

1. Click in ToC
2. Press tab to focus first setting
3. Press shift+tab until focus leaves the setting

At this point it seems like nothing is focused, from devtools:

![image](https://user-images.githubusercontent.com/2193314/78053712-d7405580-7335-11ea-9b82-2f7b62e5e194.png)

**Comment by rzhao271** [comment]
This issue should be fixed now with the use of the splitview.

**Comment by meganrogge** [comment]
maybe I'm clicking on the wrong thing, but tab always focuses the header of the section for me, which is unexpected. Further shift+tabbing does not seem to focus the settings
https://user-images.githubusercontent.com/29464607/165985077-15307e5e-e2f4-4572-ac95-bd37e87017bb.mov

**Comment by rzhao271** [comment]
@meganrogge After focus goes to a header in the Settings editor, because one is in a listview, they need to use the up/down arrow keys to navigate to the other settings.
Let me know if there's still a tabbing issue
